x-pack/filebeat/module/netflow/_meta/kibana/7/dashboard/filebeat-netflow-autonomous-systems.json

```diff
@@ -0,0 +1,597 @@
+{
+  "objects": [
+    {
+      "attributes": {
+        "description": "",
+        "hits": 0,
+        "kibanaSavedObjectMeta": {
+          "searchSourceJSON": {
+            "filter": [
+              {
+                "$state": {
+                  "store": "globalState"
+                },
+                "meta": {
+                  "alias": null,
+                  "disabled": false,
+                  "indexRefName": "kibanaSavedObjectMeta.searchSourceJSON.filter[0].meta.index",
+                  "key": "input.type",
+                  "negate": false,
+                  "params": {
+                    "query": "netflow"
+                  },
+                  "type": "phrase",
+                  "value": "netflow"
+                },
+                "query": {
+                  "match": {
+                    "input.type": {
+                      "query": "netflow",
+                      "type": "phrase"
+                    }
+                  }
+                }
+              }
+            ],
+            "highlightAll": true,
+            "version": true
+          }
+        },
+        "optionsJSON": {
+          "darkTheme": false
+        },
+        "panelsJSON": [
+          {
+            "col": 1,
+            "panelIndex": 1,
+            "panelRefName": "panel_0",
+            "row": 1,
+            "size_x": 12,
+            "size_y": 1
+          },
+          {
+            "col": 7,
+            "panelIndex": 2,
+            "panelRefName": "panel_1",
+            "row": 4,
+            "size_x": 6,
+            "size_y": 2
+          },
+          {
+            "col": 7,
+            "panelIndex": 3,
+            "panelRefName": "panel_2",
+            "row": 6,
+            "size_x": 6,
+            "size_y": 2
+          },
+          {
+            "col": 1,
+            "panelIndex": 4,
+            "panelRefName": "panel_3",
+            "row": 4,
+            "size_x": 6,
+            "size_y": 2
+          },
+          {
+            "col": 1,
+            "panelIndex": 5,
+            "panelRefName": "panel_4",
+            "row": 6,
+            "size_x": 6,
+            "size_y": 2
+          },
+          {
+            "col": 1,
+            "panelIndex": 6,
+            "panelRefName": "panel_5",
+            "row": 2,
+            "size_x": 4,
+            "size_y": 2
+          },
+          {
+            "col": 5,
+            "panelIndex": 7,
+            "panelRefName": "panel_6",
+            "row": 2,
+            "size_x": 4,
+            "size_y": 2
+          },
+          {
+            "col": 9,
+            "panelIndex": 8,
+            "panelRefName": "panel_7",
+            "row": 2,
+            "size_x": 4,
+            "size_y": 2
+          }
+        ],
+        "timeRestore": false,
+        "title": "[Filebeat] Netflow: Autonomous Systems",
```

Comment:
It should be `[Filebeat Netflow] Autonomous Systems` (same for other dashboards)